### PR TITLE
256 floatcurrency

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Fixed method
Add test case
**getFloatCurrency(191827.41133)** _// 191827.41_
**getFloatCurrency(50)** _// 50_
#### Where should the reviewer start?
/helpers/utils/getFloatCurrency
#### How should this be manually tested?
https://localhost:8080
#### What are the relevant tickets?
#256 
#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/branch-256-floatcurrency

### CODE REVIEW

File: src/helpers/utils.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ export const getMaxSum = (items, field) => {
 
 // short float currency
 export const getFloatCurrency = (n) => {
-  const value = n.toString().replace(/0+$/, '')
+  const hasPoint = () => !!~n.toString().indexOf('.')
+  const value = hasPoint() ? n.toString().replace(/0+$/, '') : n.toString()
```
Comment (roma219) [inline src/helpers/utils.js]: No need to convert it to string 3 times

**Review by roma219 — CHANGES_REQUESTED** [review]
Add a test case for
> input: 191827.41133 output: .1918274

File: src/helpers/utils.js (hunk 1 above)
Comment (lapinlapin) [reply src/helpers/utils.js]: fixed

**Comment by lapinlapin** [comment]
> Add a test case for input: 191827.41133 output: .1918274

Its incorrect case. Must begin with a point when the first number is 0

**Review by roma219 — APPROVED** [review]

**Pull request merged**